docs/get-started/quick-start-guide.md

```diff
@@ -60,84 +60,70 @@ Initialized
-...       ],
-...       "item_type": "PSOrthoTile",
-...       "product_bundle": "analytic"
-...     }
-...   ]
-... }
-...
->>> async def create_order(request):
-...     async with planet.Session() as ps:
-...         client = planet.OrdersClient(ps)
-...         return await client.create_order(request)
-...
->>> oid = asyncio.run(create_order(request))
+#### Your API Key as an Environment Variable
 
+You can also set the value of your API Key as an environment variable in your terminal at the command line:
+
+```console
+export PL_API_KEY=<your api key>
+```
+
+And you can see that the value was stored successfully as an environment variable with the following command:
+
+```console
+echo $PL_API_KEY
 ```
 
-Not into async? No problem. Just wrap the library and async operations together
-and call from your synchronous code.
+!!!note "Planet SDK uses the API Key environment variable"
+    If you do create a `PL_API_KEY` environment variable, the SDK will use this value. `PL_API_KEY` overrides the value that was retrieved from the Planet server with a call to `planet auth value`. The initial `planet auth value` won’t be updated to reflect the new value.
 
-```python
->>> def sync_create_order(order_details):
-...     return asyncio.run(create_order(order_details))
->>>
->>> oid = sync_create_order(order_details)
+## Step 5: Search for Planet Imagery
 
+You’ve installed the environment, the SDK, and connected with the Planet server. You’re now ready to get your first bunch of data.
+
+In this step, you search for the most recent PSScene images available to download and filter the list based on those images you actually have permissions to download.
+
+### planet data filter
+
+One of the commands you’ll use most frequently is `planet data filter`. This “convenience method” creates the JSON you need to run other commands. Run it with no arguments to see how it works by default:
+
+```console
+planet data filter
 ```
- 
-When using `asyncio.run` to develop synchronous code with the async library,
-keep in mind this excerpt from the
-[asyncio.run](https://docs.python.org/3/library/asyncio-task.html#asyncio.run)
-documentation:
 
-"*This function always creates a new event loop and closes it at the end. It
-should be used as a main entry point for asyncio programs, and should ideally
-only be called once.*"
+Open the `filter.json` file to see some default filters. `PermissionFilter` filters the output to only contain imagery that you have permission to download. You’ll also see `quality_category`, which means the output lists only images in the `standard quality` category. 
+
+!!!note "The --help switch is your friend"
+    You can do a lot with this `filter` command. We recommend running `planet data filter --help` often to get a reference of how the commands work.
+
+### planet data search
+
+Run the filter command and save it to a file named `filter.json`:
+
+```console
+planet data filter > filter.json
```

Comment:
I'm kinda tempted to just do the full pipe here, instead of the two step process. Like tell people to do `planet data filter | planet data search`. For someone new to the command-line both `>` and `|` will seem weird, so may as well show the bit that lets you do it all in one command. Then they won't be thinking they always have to output to a file and then input it. I think we can give a brief explanation on what is happening, and link to the 'concepts'.

Also happy for you to just merge this and I can make a PR on the 'quicker start'.

Reply:
This is the first time some readers are using the CLI, so I thought we'd break out the steps. Piping in the next module, no?

Reply:
Cool, don't feel strongly.